services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -1377,6 +1378,31 @@ class ApplicationServiceITest : DatabaseTest() {
         }
     }
 
+    @Nested
+    inner class GetAllApplicationsCreatedByUser {
+        @Test
+        fun `Returns empty list without applications`() {
+            val result = applicationService.getAllApplicationsCreatedByUser(USERNAME)
+
+            assertThat(result).isEmpty()
+        }
+
+        @Test
+        fun `Returns applications created by the user`() {
+            val hanke = createHankeEntity()
+            alluDataFactory.saveApplicationEntities(6, USERNAME, hanke) { i, application ->
+                if (i % 2 == 0) application.userId = "Other User"
+            }
+
+            val result = applicationService.getAllApplicationsCreatedByUser(USERNAME)
+
+            assertThat(result).hasSize(3)
```

Comment:
Would this: 

assertThat(userids).containsExactly(USERNAME, USERNAME, USERNAME)

also verify that the size is 3?

Reply:
Yes it does. I find it a lot simpler to see what the problem is when the size doesn't match if the size is checked separately.